packages/typespec-azure-resource-manager/lib/responses.tsp

```diff
@@ -13,7 +13,10 @@ namespace Azure.ResourceManager;
 @doc("Azure operation completed successfully.")
 model ArmResponse<ResponseBody> {
   ...OkResponse;
-  ...Body<ResponseBody>;
```

Comment:
I think we probably need a new template, BodyRoot

Reply:
Yeah i forgot about this, it's the point where BodyRoot doesn't feel like a good name